### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by skoomasteve**

### CODE REVIEW

**Comment by skoomasteve** [comment]
Just a few spelling corrections in the readme.

**Comment by Mmdixon** [comment]
Thanks! Can you fix one more, `comparisions` should be `comparisons`

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ A phonetic matching library. Includes text utilities to do string comparisons on
 Docs can be found at: https://microsoft.github.io/PhoneticMatching/
 
 Supported API:
-* C++
+* C++
```
Comment (Mmdixon) [inline README.md]: There is some tailing whitespace here that got introduced.

**Comment by skoomasteve** [comment]
No problem.
 Removed whitespace as requested; also, comparisions is now comparisons (both committed to my fork)

**Pull request merged**